src/oidcendpoint/oidc/add_on/pkce.py

```diff
@@ -96,11 +90,14 @@ def post_token_parse(request, client_id, endpoint_context, **kwargs):
         return request
```

Comment:
This is probably not strictly related to this PR but could be fixed here anyway. A check should be added here for the grant type because we don't want to check for `code` in a refresh token grant or a token exchange one.

Or we could check for the request instance like this (maybe strictly checking for the grant type to be authorization_code is cleaner tho):
```
if isinstance(
    request,
    (AuthorizationErrorResponse, RefreshAccessTokenRequest, TokenExchangeRequest),
):

```